packages/models-library/src/models_library/projects_state.py

```diff
@@ -54,15 +56,15 @@ class ProjectStatus(str, Enum):
 
 class ProjectLocked(BaseModel):
     value: bool = Field(..., description="True if the project is locked")
-    owner: Optional[Owner] = Field(
+    owner: Owner | None = Field(
         default=None, description="If locked, the user that owns the lock"
     )
     status: ProjectStatus = Field(..., description="The status of the project")
 
     class Config:
         extra = Extra.forbid
         use_enum_values = True
-        schema_extra = {
+        schema_extra: ClassVar[dict[str, Any]] = {
```

Comment:
haha ... somebody please create a PR changing all these at once :-D

Reply:
come on we can live with doing it gradually as we did for other stuff. no need to rush 🏃